qiime2/plugin/plugin.py

```diff
@@ -178,8 +347,50 @@ def decorator(validator):
         return decorator
 
     def register_transformer(self, _fn=None, *, citations=None):
-        """
-        A transformer has the type Callable[[type], type]
+        """ **Decorator** which registers a transformer to convert data
+
+        This decorator may be used with or without arguments.
+
+        Parameters
+        ----------
+        _fn : Callable
+            Ignore this parameter as it is the mechanism to allow argumentless
+            decoration
+        citations : list of CitationRecord
+          Citations to associate with a result whenever this transformer is
+          used internally. Can also use an entire :py:class:`Citations` object.
```

Comment:
these don't have the same indentition but I think that doesn't matter

Reply:
Yeah, but fixed anyway. I went with an indent of `2` above to match most of the indents.